auth-api/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>io.stargate</groupId>
         <artifactId>stargate</artifactId>
-        <version>0.0.5-SNAPSHOT</version>
+        <version>0.0.6-SNAPSHOT</version>
```

Comment:
Sorry, @dougwettlaufer , could you give a TL;DL on why the previous approach of using `${revision}` for version properties did not work? thx!

Reply:
Using the property for building/running worked because maven substituted the value on the fly but when it came time to publish to GH packages instead of replacing `${revision}` with `0.0.1` it was instead keeping the variable and using that in the deployed pom. So whenever you pulled in the dependency into another project it broke saying `${revision} doesn't exist`.